modules/installation-configuration-parameters.adoc

```diff
@@ -1032,6 +1032,72 @@ If you deploy to a custom subnet, you cannot specify an external DNS server to t
+====
+// Undefine {FeatureName} attribute, so that any mistakes are easily spotted
+:!FeatureName:
+
+{rh-openstack-first} deployments do not have a single implementation of failure domains. Instead, availability zones are defined individually for each service, such as the compute service, Nova; the networking service, Neutron; and the storage service, Cinder. 
+
+Beginning with {product-title} 4.12, there is a unified definition of failure domains for {rh-openstack} deployments that covers all supported availability zone types. You can use failure domains to control related aspects of Nova, Neutron, and Cinder configurations from a single place. 
+
+In {rh-openstack}, a port describes a network connection and maps to an interface inside a compute machine. A port also:
+* Is defined by a network or by one more or subnets
+* Connects a machine to one or more subnets
+
+Failure domains group the services of your deployment by using ports. If you use failure domains, each machine connects to:
+* The `portTarget` object with the ID `control-plane`.
+* All non-control-plane `portTarget` objects within its own failure domain.
+* All networks in the machine pool's `additionalNetworkIDs` list.
+
+To configure failure domains for a machine pool, edit availability zone and port target parameters under `controlPlane.platform.openstack.failureDomains`.
+
+.{rh-openstack} parameters for failure domains
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.openstack.failuredomains.computeAvailabilityZone`
+|An availability zone for the server. If not specified, the cluster default is used.
+|The name of the availability zone. For example, `nova-1`.
+
+|`platform.openstack.failuredomains.storageAvailabilityZone`
+|An availability zone for the root volume. If not specified, the cluster default is used.
+|The name of the availability zone. For example, `cinder-1`.
+
+|`platform.openstack.failuredomains.portTargets`
+|A list of `portTarget` objects, each of which contextualizes a port target within a single failure domain. Each object must include an `id` parameter that specifies a port target. Optionally, you can include other parameters.
+|A list of `portTarget` objects.
+
+|`platform.openstack.failuredomains.portTargets.portTarget.id`
+|The ID of an individual port target. Set the value of this parameter to `control-plane` to select that port target as the first network for machines. If this parameter is not set, the cluster default network is used.
+|`control-plane` or an arbitrary string.
+
+|`platform.openstack.failuredomains.portTargets.portTarget.network`
+|Required. The name or ID of a network to use in place of the primary cluster network.
+|A network object that contains a name or UUID. For example, `8db6a48e-375b-4caa-b20b-5b9a7218bfe6`.
+
+|`platform.openstack.failuredomains.portTargets.portTarget.fixedIPs`
+|Subnets to to allocate fixed IP addresses to. These subnets must within the same network as the port.
```

Comment:
be? exist? It sounds like a verb is missing here

Reply:
```suggestion
|Subnets to allocate fixed IP addresses to. These subnets must within the same network as the port.
```

Reply:
>These subnets must within the same network as the port.

Something is still off

Reply:
Must've missed that, somehow. TY.

Reply:
@pierreprinetti Actually, I think you're just commenting on an outdated file. 

See most recent commit: https://github.com/openshift/openshift-docs/pull/57526/commits/57e662dbf995d127e57e8ccd11e2611547f040c8